projects/plugins/jetpack/extensions/extended-blocks/index.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
I'm not sure I understand why this needs to be in `extended-blocks`? Wouldn't this make more sense to be in a more generic location, since it's used by all kinds of blocks?

Reply:
I've moved the code straight into the edtior.js file. I think it's a good place to put the code. c68ac30907698a2fe934eb62751e9870f7287c4c